### ISSUE TITLE
Evaluating accuracy of the model

### ISSUE THREAD

**Issue by HananAbdulla** [Nikolai10/scrabble-gan#15]
Hello!
@Nikolai10

When using deep learning, the dataset is divided into training set, validation set and testing set.
I notice that there are not folders for these division!
Does that mean GAN approach has another way, and how we evaluate accuracy of the model? 

Thanks..

**Comment by Nikolai10** [comment]
Hello @HananAbdulla, 

details can be found in the official [paper](https://arxiv.org/pdf/2003.10557.pdf), section 4.2, 4.3.
